src/main/java/frc/robot/Controls.java

```diff
@@ -115,7 +120,23 @@ private void configureArmPivotBindings() {
 
     // Arm Controls binding goes here
     s.armPivotSubsystem.setDefaultCommand(
-        s.armPivotSubsystem.startMovingVoltage(() -> Volts.of(6 * operatorController.getLeftY())));
+        s.armPivotSubsystem
+            .startMovingVoltage(() -> Volts.of(6 * secretThirdController.getLeftY()))
```

Comment:
By default, the arm shouldn't be moving. I'm not sure the arm even needs a default command, as by default it should either keep position (if the last command was `moveToPosition`) or just stop (if the last command was `startMovingVoltage`)

Reply:
And recommend adding buttons to manually move the arm with positive & negative values for `startMovingVoltage`